anchore_engine/services/apiext/swagger/swagger.yaml

```diff
@@ -2585,6 +2585,32 @@ paths:
           description: Internal error
           schema:
             $ref: "#/definitions/ApiErrorResponse"
+  /system/webhooks/{webhook_type}/test:
```

Comment:
Hey @dakaneye, good to see so many PRs and fixes! Quick question about this new API, is there a particular reason it's nested under `/system`? I generally think of that path as ops internal to the system such as services, events etc. I think webhook is user facing enough to merit its own endpoint, worth the discussion

Reply:
Hey @nightfurys ! I originally based the design as you had mentioned: `/webhooks/test/{webhook_type}`, but I updated it based on the comment that @zhill made [here](https://github.com/anchore/anchore-engine/pull/569). I'm happy to make it it's own base path though, depending on the team's preference. I think with the other enhancement issue, to expose a schema, an API may be a good way. But also, as Zach commented, the schema of a webhook shouldn't be changing statically, so it may be best to expose it as a swagger definition... 

Let me know what sounds best!